k8s/production/kube-system/gp3-storage-class.yaml

```
@@ -0,0 +1,11 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: gp3
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/aws-ebs
```

This provisioner is no longer supported (see [the docs](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi-migration-faq.html#csi-migration-faq-aws-ebs-provisioner)). The new way is to use the Amazon EBS CSI driver - see https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html.

What do you think about migrating to that while we're here? Since we have the EBS CSI driver installed, it looks like the only change needed for that is
```suggestion
provisioner: ebs.csi.aws.com
```